applications/machine_learning/sample.yaml

```
@@ -37,3 +66,13 @@ tests:
       - thingy53/nrf5340/cpuapp/ns
     tags: ci_build
     extra_args: CONF_FILE=prj_release.conf
+  applications.machine_learning.zrelease.sysbuild:
+    sysbuild: true
+    build_only: true
+    platform_allow: nrf52840dk/nrf52840 thingy53/nrf5340/cpuapp thingy53/nrf5340/cpuapp/ns
+    integration_platforms:
+      - nrf52840dk/nrf52840
+      - thingy53/nrf5340/cpuapp
+      - thingy53/nrf5340/cpuapp/ns
+    tags: ci_build sysbuild
+    extra_args: CONF_FILE=prj_release.conf
```

Shouldn't we use `FILE_SUFFIX`?
```suggestion
    extra_args: FILE_SUFFIX="release" 
```
Also in other places.

---

Fixed